client/rpc-api/src/author/error.rs

```diff
@@ -25,6 +25,9 @@ pub type Result<T> = std::result::Result<T, Error>;
 /// Author RPC future Result type.
 pub type FutureResult<T> = Box<dyn rpc::futures::Future<Item = T, Error = Error> + Send>;
 
+/// Author RPC future Result type with specific lifetime.
+pub type FutureResultLt<'a, T> = Box<dyn rpc::futures::Future<Item = T, Error = Error> + Send + 'a>;
```

Comment:
I think it would be better to do:
```
pub type FutureResult<'a, T> = Box<dyn Future<...> + Send + 'a>;

pub type StaticFutureResult<T> = FutureResult<'static, T>;
```
The `Lt` prefix does not seem intuitive to me.

Reply:
This result with lifetime was not used already, so just deleted it